src/web/client/schema/portalSchema.ts

```diff
@@ -80,13 +80,14 @@ export const portal_schema_V1 = {
                 _fetchQueryParameters:
                     "?$filter=adx_webpageid eq {entityId} &$select=adx_name,adx_copy,adx_customcss,adx_customjavascript,adx_partialurl,_adx_webpagelanguageid_value&$count=true",
                 _multiFileFetchQueryParameters:
-                    "?$filter=_adx_websiteid_value eq {websiteId} and _adx_webpagelanguageid_value ne null &$select=adx_webpageid,_adx_webpagelanguageid_value,adx_name,adx_copy,adx_customcss,adx_customjavascript,adx_partialurl&$count=true",
+                    "?$filter=_adx_websiteid_value eq {websiteId} and _adx_webpagelanguageid_value ne null &$select=adx_webpageid,_adx_webpagelanguageid_value,adx_name,adx_copy,adx_customcss,adx_customjavascript,adx_partialurl,_adx_rootwebpageid_value&$count=true",
                 _attributes: "adx_customcss,adx_customjavascript,adx_copy",
                 _attributesExtension: new Map([
                     ["adx_customcss", "customcss.css"],
                     ["adx_customjavascript", "customjs.js"],
                     ["adx_copy", "webpage.copy.html"],
                 ]),
+                _rootwebpageid: "_adx_rootwebpageid_value",
```

Comment:
same for this change - we do not need to add _rootwebpageid here. Just fetch it in the fetchQueryparam calls and use it after reading this value from constants.

Also - you need to add it for both _fetchQueryParameters and _multiFileFetchQueryParameters. We support single file and multi file view of files still.

Reply:
_rootwebpageid is added to get the attribute name, didn't want to make it confusing by adding with _attributes field, which has file extension